backend/game/cards/discard_cards/victory_point_manipulation_cards/fire_blast.py

```diff
@@ -0,0 +1,11 @@
+from game.cards.card import Card
+from game.cards.discard_card import DiscardCard
+
+
+class FireBlast(DiscardCard):
+    def __init__(self):
+        Card.__init__(self, "Fire Blast", 3, "All other monsters - 2[health]", "")
+
+    def immediate_effect(self, player_that_bought_the_card, other_players):
+        for other_player in other_players:
+            other_player.update_energy_by(-2)
```

Comment:
Why is energy being changed here? The card is about damaging other players.

Reply:
Good catch, made the update to health and pushed in latest commit